plugins/Jobs.cpp

```diff
@@ -638,3 +679,20 @@ string BedrockPlugin_Jobs::_constructNextRunDATETIME(const string& lastScheduled
     // Combine the parts together and return the full DATETIME statement
     return "DATETIME( " + SComposeList(safeParts) + " )";
 }
+
+/**
+ * Returns the number of RUNNING or QUEUED jobs that have the passed jobID as their parent.
+ */
+int BedrockPlugin_Jobs::_getCountOfPendingChildJobs(SQLite& db, const int jobID)
+{
+    SQResult result;
+    if (!db.read("SELECT count(1) "
```

Comment:
There's actually no reason to return a full count -- that is making it do extra work to produce an unnecessarily specific result.   How about instead just `SELECT 1 FROM ... WHERE ... LIMIT 1;` -- this way it'll return the instant it finds one result, rather than counting up all the results just so we can log an interesting number.

So, I suggest changing this to `_checkForPendingChildJobs()` and have it just return true/false.  This also simplifies the code that calls this function, as it eliminates the need to store this unnecessary result into an unnecessary variable.

Finally, don't use `const int jobID`.  The `const` is unnecessary, and the `int` should be an `int64`.